### DISCUSSION TITLE
[Bug] 链接导致页面样式错误

### DISCUSSION THREAD

**Discussion by I21b** [travellings-link/travellings#2495] [寻求帮助] [answered]
## 描述
有网站直接链接到gobyclouds页面，导致绕过设置样式错误，让深色模式满亮度眼睛爆炸

## 可能的解决方法
所有样式的链接都先读一遍cookies重定向再显示

**Comment by BLxcwg666** [comment, marked as answer]
如果站长直接链接到某个页面的话我们可能无权干涉  
不过这样解决好像也没有问题  
可以等组内再讨论一下
